### ISSUE TITLE
Target NETStandard

### ISSUE THREAD

**Issue by morquan** [HelloFax/hellosign-dotnet-sdk#32]
Greetings! I'm evaluating integrating Hellosign into a new solution. Are there plans to target NETStandard to support those of us running on .NET Core?

**Comment by luricky** [comment]
Any plan on supporting .NET core?

**Comment by morquan** [comment]
I've been told they have no plans. I did find this unofficial implementation: https://github.com/soukoku/HelloSignApi

**Comment by luricky** [comment]
Thanks

**Comment by alexmac05** [comment]
At HelloSign, we do agree that this would definitely be nice to have. The current obstacle is that the SDK is based on the RestSharp framework, which isn't fully compatible with .NET Core.

Do you two happen to know of a version/fork of RestSharp that is compatible with .NET Core? If so, let us know. That is the holdup. 

Thank you!

**Comment by luricky** [comment]
Seems like RestSharp just updated version 106.1.0 to support .net core 2.0

Ricky Lu

**Comment by alexmac05** [comment]
If you all had to choose a priority level for this request, could you let me know where you all are at with this one?

1. Deal Breaker
2. An important enhancement for us
3. Nice to have but not top of mind
4. We have already found a workaround and no longer are concerned with this

Thank you,
alex m. 
internal tracking number AS-54

**Comment by pheuter** [comment]
2.

Looking for an e-signature solution to get started with ASAP for our startup, coming down to Docusign vs HelloSign. Docusign is currently lagging on their .netstandard2.0 support. If this could be made available we'll likely go with HelloSign.

**Comment by genne** [comment]
Any updates on this? We're trying to migrate our solution to .NET Core, but are currently blocked by this.

**Comment by THammond9** [comment]
Are there any updates for this?

**Comment by BHSPitMonkey** [comment]
Update: Apologies for the long wait times involved in making this happen, but this is nearly finished and we'll be publishing a new release soon that will target .NET Standard 2.0 and .NET Framework 4.5.2! Will update and close this issue once it's available.

**Comment by BHSPitMonkey** [comment]
This is now supported as of v1.0.0! Take a look at the [Releases page](https://github.com/hellosign/hellosign-dotnet-sdk/releases) for all the details. Thanks for everyone's input and patience on this!
